src/app/helptext/storage/volumes/datasets/dataset-form.ts

```diff
@@ -168,6 +168,13 @@ dataset_form_encryption: {
     key_tooltip: T('Enter or paste a string to use as the encryption key for this dataset.'),
     key_validation: [Validators.minLength(64), Validators.maxLength(64)],
     non_encrypted_warning_title: T('Warning'),
-    non_encrypted_warning_warning: T('Disabling encryption will result in a non-encrypted dataset, continue?')
-}
+    non_encrypted_warning_warning: T('Disabling encryption will result in a non-encrypted dataset, continue?'),
```

Comment:
This reads circular or redundant (disable encryption to disable encryption). Suggestion focuses on the result of the action first:
```suggestion
    non_encrypted_warning_warning: T('All data stored in this dataset will be decrypted and the dataset marked \
 as non-encrypted. Do you want to continue?'),
```